components/youtube_analytics_api/actions/list-channel-reports/list-channel-reports.mjs

```
@@ -26,16 +26,19 @@ export default {
+        constants.CHANNEL_REPORT_TYPE;
 
       if (channelReportType === VIDEO_BASIC_USER_ACTIVITY_STATS.value) {
-        const supportedFilters = VIDEO_BASIC_USER_ACTIVITY_STATS.metadata.filters
-          .reduce((acc, filter) => ({
-            ...acc,
-            [filter]: "",
-          }), {});
+        const supportedFilters =
+          VIDEO_BASIC_USER_ACTIVITY_STATS.metadata.filters.reduce(
+            (acc, filter) => ({
+              ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)